crmsh/bootstrap.py

```diff
@@ -2173,9 +2177,16 @@ def bootstrap_join(context):
         lock_inst = lock.RemoteLock(cluster_node)
         try:
             with lock_inst.lock():
+                _context.node_list_in_cluster = utils.fetch_cluster_node_list_from_node(cluster_node)
```

Comment:
Key point is at the beginning of cluster join, get the list of node in cluster from init node, assign to `_context.node_list_in_cluster`, which will be used in `sync_file`